UI/frontend-plugins/frontend-tools/data/scripts/url-text.py

```diff
@@ -15,8 +15,9 @@ def update_text():
 
 	source = obs.obs_get_source_by_name(source_name)
 	if source is not None:
+		obs_user_agent = "obs-studio/26.0.2 (Macintosh; Intel Mac OS X 10_12) Python/3.8.5"
```

Comment:
I'm not a fan of this, though I assume it's only included because it's required.

Reply:
It's included because e.g. Cloudflare will block all requests by the script otherwise and this way at least it doesn't pretend to be a browser. You're right that the OS and Python version could be made dynamic, I can try and make this dynamic real quick..

Reply:
Maybe it'd be worth as a separate change/PR where we generate default `urllib` headers (for all scripts) in the c code itself? Sort of like we configure logging:

https://github.com/obsproject/obs-studio/blob/3564e446ffa076ace93bb1f69f3dcddc93025b6e/deps/obs-scripting/obs-scripting-python.c#L32-L48

https://github.com/obsproject/obs-studio/blob/3564e446ffa076ace93bb1f69f3dcddc93025b6e/deps/obs-scripting/obs-scripting-python.c#L1673

That way it's not scoped to one built-in script.

Reply:
Fair, I can remove it from this PR. If I'm not mistaken to achieve this we'd have to import `urllib` ourselves (imports are "singletons" in Py) then monkey patch its internals to use our customised headers. Otherwise scripts would need to add these headers to their `urllib` requests manually.